docs/runbooks/DeprecatedRHEL6Vm.md

```diff
@@ -0,0 +1,28 @@
+# DeprecatedRHEL6Vm
+
+## Meaning
+
+This alert fires when a running virtual machine is based on RHEL6 template, which will not be supported in the next release.
+
+## Impact
+RHEL6 virtual machines will not be supported in the next release in case of issues.
+
+## Diagnosis
+
+1. Check which running virtual machines has the rhel6 template label.
+
+   ```bash
+   $ kubectl get -A vm -ojson | jq  '.items | map(select((.metadata.labels."vm.kubevirt.io/template" | test("rhel6-.*")?) and .status.printableStatus == "Running")) | {kind: "List", items: .}'  | kubectl get -A -f -
+   ```
+
+## Mitigation
+
+Upgrade all the RHEL6 virtual machines to a newer version or stop them.\
+Since the warning looks at the value of the `vm.kubevirt.io/template` label and is triggered if it starts with `rhel6`,
+in case of an update, you should also manually change its value to a correct value other than `rhel6-*`.
```

Comment:
@fossedihelm  Can this be done by a command? If a customer has hundreds ov VMs, does this resolution makes sense?

Reply:
@sradco Well, regarding the upgrade it has to be done manually. We can provide a command that will patch the vm label. in this case I tend to opt for the deletion of the label itself, since we cannot know to which version the vm will be updated.
WDYT? Thanks

Reply:
Deleting the label will prevent having a similar alert for RHEL7 and above when needed..